portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityTransferServiceProviderRegistry.java

```diff
@@ -84,4 +86,26 @@ public PortabilityTransferServiceProviderRegistry(List<String> enabledServices,
     Preconditions.checkArgument(serviceProvider != null);
     return serviceProvider.getImporter(transferDataType);
   }
+
+  /**
+   * Returns the set of service ids that can transfered for the given {@code transferDataType}.
+   *
+   * @param transferDataType the transfer data type
+   */
+  @Override
+  public Set<String> getServices(String transferDataType) {
+    Preconditions.checkArgument(supportedExportTypes.contains(transferDataType),
+        "TransferDataType [%s] is not valid for export", transferDataType);
+    Preconditions.checkArgument(supportedImportTypes.contains(transferDataType),
+        "TransferDataType [%s] is not valid for import", transferDataType);
+    return serviceProviderMap.keySet();
```

Comment:
I was wondering the same thing. think the goal would be each exportable service also has an importer and vice versa so I wanted to avoid making the code that flexible and allowing for inconsistent services.

Reply:
Also, moved these methods to the AuthServiceProviderRegistry